_posts/2024-10-01-understanding-index-request-cache.markdown

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Optimizing query performance: The implementation of index request cache in OpenSearch" 
+authors:
+   - awskiran
+   - kkhatua
+   - upasagar
+   - kolchfa
+date: 2024-10-01 
+categories:
+  - technical-posts
+  - search
+meta_keywords: OpenSearch cluster, Opensearch caching, index request cache, search performance optimization, search latency
+meta_description: 
+---
+
+Speed and efficiency are essential for search users. OpenSearch achieves these through various mechanisms, one of the most important of which is [index request cache](https://opensearch.org/docs/latest/search-plugins/caching/request-cache/). This blog post describes how this cache is implemented and how it optimizes query performance.
+
+## What is index request cache?
+
+Index request cache is designed to accelerate search queries in OpenSearch by storing query results at the shard level, as shown in the following image. 
+
+![Index-Request-Cache](/assets/media/blog-images/2024-10-01-understanding-index-request-cache/cache_location.png){:class="img-centered" style="width:750px;"}
```

Comment:
In this image, please change "Opensearch" to "OpenSearch".

Reply:
Done, good call !